src/systems/filecoin_mining/storage_mining/storage_miner_actor.go

```diff
@@ -109,25 +99,50 @@ func (a *StorageMinerActorCode_I) NotifyOfPoStChallenge(rt Runtime) InvocOutput
 }
 
 func (st *StorageMinerActorState_I) _updateCommittedSectors(rt Runtime) {
-	for sectorNo, sealOnChainInfo := range st.StagedCommittedSectors() {
-		st.Sectors()[sectorNo] = sealOnChainInfo
+	for sectorNo, stagedInfo := range st.StagedCommittedSectors() {
```

Comment:
the name `_updateCommittedSectors` is not clear enough. maybe `_unstage` or `_moveCommittedSectorsToProvingSet` or something clear like this but shorter. maybe `_processStagedCommittedSectors`